app/views/external_users/claims/supporting_evidence_checklist/_fields.html.haml

```diff
@@ -1,7 +1,13 @@
-%fieldset.evidence-checklist{ 'data-mute-indictment': @claim.allows_fixed_fees? ? 'true' : 'false' }
-  %legend.form-label-bold
-    = t('.supporting_evidence_checklist')
-    %span.form-hint
-      = t('.supporting_evidence_checklist_hint')
+= f.govuk_check_boxes_fieldset :evidence_checklist_ids,
+  classes: 'cc-multiple-choice',
```

Comment:
This class wraps the group, not the individual check boxes, therefore i don't think this is doing what you expected.